scripts/gulp/tasks/bundle.ts

```diff
@@ -53,7 +44,14 @@ task('bundle:package:types:tsc', () => {
 task('bundle:package:types:copy', () => {
   return src(paths.packageDist(packageName, 'dts/src/**/*.d.ts')).pipe(dest(paths.packageDist(packageName, 'es')));
 });
-task('bundle:package:types', series('bundle:package:types:tsc', 'bundle:package:types:copy'));
+task('bundle:package:types:clean', () => {
+  return del([`${paths.packageDist(packageName)}/dts`], { force: true });
+});
+
+task(
+  'bundle:package:types',
+  series('bundle:package:types:tsc', 'bundle:package:types:copy', 'bundle:package:types:clean'),
```

Comment:
Types should exist only under `es` directory, this will remove temporary `dts` directory after build.